core/controllers/editor_test.py

```diff
@@ -1381,6 +1391,49 @@ def test_for_assign_role_for_exploration(self):
         self.assert_cannot_voiceover(exp_id)
         self.logout()
 
+        # Check that COLLABORATOR 4 who cannot edit but can voiceover,
+        # cannot add a new State 6.
+        self.login(self.COLLABORATOR4_EMAIL)
+        csrf_token = self.get_new_csrf_token()
+
+        # Since, check edit_activity_always return false, therefore
+        # editor will behave as voiceover artist.
```

Comment:
```suggestion
        # user will behave as voiceover artist.
```

Since the user in this case isn't an editor

Reply:
committed this suggestion